std/src/main/scala/algebra/std/map.scala

```diff
@@ -130,3 +133,25 @@ class MapRng[K, V](implicit V: Rng[V]) extends MapSemiring[K, V] with Rng[Map[K,
   override def minus(x: Map[K, V], y: Map[K, V]): Map[K, V] =
     subtractMap(x, y)(V.minus)(V.negate)
 }
+
+class MapGenBool[K, V](implicit GBV: GenBool[V], EqV: Eq[V]) extends GenBool[Map[K, V]] {
```

Comment:
Yes, `MapVectorEq` is the equivalence in use, and there is an instance of `AdditiveMonoid[Set[Byte]]` (not group), namely the `SetSemiring` in `std/set.scala`.

Btw, it looks like `MapEq` is not used anywhere, just `MapVectorEq`.

Regarding whether some type classes should define their own `Eq`, I was wondering the same, but @non [thought otherwise](https://gitter.im/non/cats?at=56423db104a883da3838e0bf), and I tend to agree now.